libretro.cpp

```diff
@@ -569,9 +571,8 @@ static bool LoadCommon(std::vector<CDIF *> *CDInterfaces)
 
  if(!BRAMDisabled)
  {
-  // Initialize backup RAM
-  memset(BackupRAM, 0, sizeof(BackupRAM));
-  memset(ExBackupRAM, 0, sizeof(ExBackupRAM));
+  // Initialize Save RAM
```

Comment:
not sure it matter in this codebase but I think comments are prefered as /* */ blocks

Reply:
i think cpp accepts // as comments. will whole block the others though.